src/tabs/tabs.scss

```diff
@@ -177,38 +177,41 @@
       background-color: t(iui-color-foreground-positive);
     }
   }
-
-  ~ .iui-tab-stripe {
-    display: none;
-  }
 }
 
 @mixin iui-tab-animated {
-  ~ .iui-tab-stripe {
+  &::after {
     position: absolute;
+    content: ' ';
     @include themed {
       background-color: t(iui-color-foreground-primary);
     }
   }
 
-  &.iui-green ~ .iui-tab-stripe {
+  &.iui-green::after {
     @include themed {
       background-color: t(iui-color-foreground-positive);
     }
   }
 
   @at-root {
-    .iui-horizontal & ~ .iui-tab-stripe {
+    .iui-horizontal &::after {
+      top: calc(100% - #{$iui-xxs});
       height: $iui-xxs;
+      left: var(--stripe-left);
+      width: var(--stripe-width);
       @media (prefers-reduced-motion: no-preference) {
         transition: width $iui-speed-fast ease-out, left $iui-speed-fast ease-out;
       }
     }
   }
 
   @at-root {
-    .iui-vertical & ~ .iui-tab-stripe {
+    .iui-vertical &::after {
+      left: calc(100% - #{$iui-xxs});
       width: $iui-xxs;
+      top: var(--stripe-top);
```

Comment:
And it won't work on IE.

Reply:
Yes. For IE, we have `iui-not-animated`

Reply:
We will check whether CSS variables are supported and set that class then?

Reply:
In React? Yeah we can do that. Or we could leave it upto the developer

Reply:
Sure we can leave, but then.. I do feel we need to start thinking about either readme for every component or something else.. Cause going through source codes and trying to figure this out is really not a nice experience.

Reply:
[Only 7 months of life support left on IE11.](https://death-to-ie11.com/)  But seems like we have the not animated class as an alternative solution.